app/controllers/admin/venues_controller.rb

```diff
@@ -19,13 +19,15 @@ def update
   def create
     @venue.update(status: "online")
     @venue.admin.roles << Role.find_by_name("venue_admin")
+    UserNotifierMailer.send_application_outcome_email(@venue, true).deliver
```

Comment:
@matthewrpacker my vote is for this to live in a private method in the controller. @GSmes what do you think?